homeassistant/components/device_tracker/luci.py

```diff
@@ -11,25 +11,27 @@
 from datetime import timedelta
 
 import requests
+import voluptuous as vol
 
-from homeassistant.components.device_tracker import DOMAIN
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.device_tracker import DOMAIN, PLATFORM_SCHEMA
 from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME
-from homeassistant.helpers import validate_config
 from homeassistant.util import Throttle
 
 # Return cached results if last scan was less then this time ago.
 MIN_TIME_BETWEEN_SCANS = timedelta(seconds=5)
 
 _LOGGER = logging.getLogger(__name__)
 
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_HOST): cv.string,
```

Comment:
To extend you need to access the `.schema` attribute
 
```python
PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
  OTHER_SCHEMA.schema)
```

Instead of this I created a shortcut for all `str` schemas, which results in a very compact form. Comments welcome